src/down.ml

```diff
@@ -994,7 +994,7 @@ let down_readline p =
 external sigwinch : unit -> int = "ocaml_down_sigwinch"
 let install_sigwinch_interrupt () =
   (* Sufficient to interrupt the event loop on window size changes. *)
-  Sys.set_signal (sigwinch ()) (Sys.Signal_handle (fun _ -> ()))
+  if Sys.unix then Sys.set_signal (sigwinch ()) (Sys.Signal_handle (fun _ -> ()))
```

Comment:
There are no signals on Windows. But there is a way to receive windows resize events which could be piped back up to the OCaml code; it would just need a bit of work.